README.md

```diff
@@ -1,2 +1,4 @@
 # developers-2015-11-23
 GitHub for Developers (November 23-24, 2015)
+
```

Comment:
Your bio file looks :sparkles: @stevensgill!  You might want to remove these lines from the README file before merging, but it's definitely not a deal breaker. :+1: